packages/app/cypress/e2e/cypress-in-cypress-run-mode.cy.ts

```diff
@@ -92,13 +84,3 @@ describe('Cypress In Cypress - run mode', { viewportWidth: 1200 }, () => {
     cy.findByTestId('sidebar').should('not.exist')
   })
 })
-
-function simulateRunModeInUI () {
```

Comment:
This always felt hacky, it was OK prior to Dec 2, 2022, but the new approach is a lot more stable than modifying the window since the value will be there before the app initializes at all.